invenio.cfg

```diff
@@ -277,6 +277,11 @@ CDS_EOS_OFFLOAD_X509_KEY_PATH = ""
 # check nginx config for more details
 CDS_EOS_OFFLOAD_REDIRECT_BASE_PATH = ""
 
+# CDS Migration
+CDS_REDIRECTION_COLLECTIONS_MAPPING = {
+    "CERN Student Project Notes": "", # uuid
```

Comment:
is it set on openshift? or we expect to set it when we have the ready uuid?

Reply:
We should set it when we have it ready

Reply:
Btw the collection name in the URL is `CERN Students Projects`. Can you fix that?